src/visualizations/timeline.js

```diff
@@ -17,24 +17,23 @@ var time = require("../util/time.js");
  * titleinfo-container (visible titleinfo)
  * */
 
+// Helper functions used when hover state changes.
+function set_color(elem_id, color) {
```

Comment:
Is it possible for the on-click events to call these? If so, awesome!
The reason why I inserted these functions with a <script> element was because i thought they would be out of scope otherwise.